app/controllers/follows_controller.rb

```diff
@@ -0,0 +1,23 @@
+class FollowsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :set_question, only: [:create, :destroy]
+
+  authorize_resource
```

Comment:
No, I download @follow before action destroy...I have a small problem with action create, because author shouldn't create @follow to his question